ui/src/dashboards/actions/index.ts

```diff
@@ -16,7 +16,7 @@ import {
   updateView as updateViewAJAX,
 } from 'src/dashboards/apis'
 import {createDashboardFromTemplate as createDashboardFromTemplateAJAX} from 'src/templates/api'
-
+import * as api from 'src/client'
```

Comment:
be explicit here. it's a contract between modules

Reply:
but convention